docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -602,7 +602,60 @@ to the child entities based on the newly created parent entity's ID.
 
 {% endinfo_block %}
 
-#### Error codes
+## Non-transactional saving
+
+By default, the Data Exchange API uses a transactional approach to save data.
+This means that if an error occurs during the saving process, the entire transaction is rolled back, and no data is saved. 
+However, in some cases, you may want to save data non-transactionally. In no-transactional mode, the API wraps each entity and its related records (if present in the request) in a separate transaction.
```

Comment:
Please mention how exactly the data will be saved, and what will not be saved in this case.